cdc/sink/codec/json.go

```diff
@@ -38,7 +38,7 @@ const (
 	// DefaultMaxMessageBytes sets the default value for max-message-bytes
 	DefaultMaxMessageBytes int = 64 * 1024 * 1024 // 64M
 	// DefaultMaxBatchSize sets the default value for max-batch-size
-	DefaultMaxBatchSize int = 4096
+	DefaultMaxBatchSize int = 16
```

Comment:
Better to add some comments, about why `16` is used

Reply:
I don't know if this is a suitable value, but `4096` is too long